src/e2e-tests/features/autelis.feature

```diff
@@ -0,0 +1,96 @@
+@autelis
+Feature: RoboDomo Autelis page
+
+
+    @S1 @automated
+    Scenario Outline: Check Autelis pool/spa/solar options
+        Given User loads the RoboDomo web app
+        When User clicks Autelis menu button
+        Then User is redirected to Autelis page
+        Then Autelis tab shall be selected
+        When User clicks on Autelis tab
+        When User clicks on <offButton> button on Autelis page
+        Then All Autelis SPA-POOL parameters should be turned off
+        When User clicks on <poolButton> button on Autelis page
+        Then Pool parameters are displayed
+        When User clicks on <spaButton> button on Autelis page
+        Then Spa parameters are displayed
+        And Solar parameters are displayed
+        Examples:
+            | offButton  | poolButton | spaButton  |
+            | OFF        | POOL       | SPA        |
+
+    @S1 @automated
+    Scenario Outline: Check Solar, Cleaner, Pool Heat, Pool light, Waterfall options 'On' and 'Off'
```

Comment:
Same for this Scenario as the above. Please decide on a common or independent approach. If independent then please remove the Outline.

Reply:
Updated